generate.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-protoc --gofast_out=. agent_payload.proto
+protoc --proto_path=../../../:. --gogofast_out=. agent_payload.proto
```

Comment:
as discussed, let's change `--proto-path` to `$GOPATH/src:.`

(same thing below)